src/core/include/openvino/core/parallel.hpp

```diff
@@ -121,11 +124,12 @@ inline int get_max_nested_levels() {
 // Controls the number of nested parallel blocks.
 // This flag has higher priority than pragma num_threads.
 inline void set_max_nested_levels(int levels) {
-#    if defined(_MSC_VER) && !defined(__INTEL_COMPILER)
-    return omp_set_nested(levels);
-#    else
+#    if _OPENMP < 201811
```

Comment:
Should the `_OPENMP` be checked if defined at all?

Reply:
Check added